packages/fxa-react/components/AppErrorBoundary/index.test.tsx

```diff
@@ -3,26 +3,16 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { render } from '@testing-library/react';
-import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
 import AppErrorBoundary from '.';
 
-describe('AppErrorBoundary', () => {
-  beforeEach(() => {
-    // HACK: Swallow the exception thrown by BadComponent
-    // it bubbles up unnecesarily to jest and makes noise
-    jest.spyOn(console, 'error');
-    (global.console.error as jest.Mock).mockImplementation(() => {});
-  });
-
-  afterEach(() => {
-    (global.console.error as jest.Mock).mockRestore();
-  });
+const { queryByTestId } = screen;
```

Comment:
If we need to `rerender`, could this change? I'm not sure this should be globally set here. Maybe just update `queryByTestId` references to `screen.queryByTestId`?